hs_core/views/utils.py

```diff
@@ -615,12 +615,6 @@ def get_my_resources_list(user):
     # we won't hit the DB for each resource to know if it's status is public/private/discoverable
     # etc
     resource_collection = resource_collection.select_related('raccess')
-    # prefetch all the metadata elements used in my resources page - no need to prefetch
-    # 'content_object' by itself
-    resource_collection = resource_collection.prefetch_related('content_object__creators')
```

Comment:
These prefetches are problematic as they work only if all the 'content_object's are of the same contenttype. Once we convert all resources to composite resource type we should be able to use prefetch of 'content_object' .